etc/containerpilot.json

```diff
@@ -6,7 +6,7 @@
   },
   "services": [
     {
-      "name": "mysql",
+      "name": "{{ if .SERVICE_NAME }}{{ .SERVICE_NAME }}{{ else }}mysql{{ end }}",
```

Comment:
I think we also need to update the `backends->name` field below so that we'll be looking for the same primary.